doc/user/bgp.rst

```diff
@@ -2947,14 +2947,16 @@ This group of server links is referred to as an Ethernet Segment.
 Ethernet Segments
 """""""""""""""""
 An Ethernet Segment can be configured by specifying a system-MAC and a
-local discriminator against the bond interface on the PE (via zebra) -
+local discriminator or a complete ESINAME against the bond interface on the
+PE (via zebra) -
 
-.. clicmd:: evpn mh es-id (1-16777215)
+.. clicmd:: evpn mh es-id [(1-16777215)|ESINAME]
```

Comment:
I believe it's not an optional parameter? It's optional for NO form, right?

Reply:
Optional for both forms.
Two style: one is `sysmac+id`, the other is `ESINAME`. 
If the one after `es-id` is 1-byte, it is the first style, if 10-byte it is the second style.

Reply:
IMO we should add a new command, like evpn mh es-name <ESINAME>

Reply:
I mean it's not possible to enter just a bare command like `evpn mh es-id` right? According to this documentation change, it seems id or name are optional.

Reply:
> It's optional for NO form, right?

Yes.


> I mean it's not possible to enter just a bare command like `evpn mh es-id` right? According to this documentation change, it seems id or name are optional.

Yes, it will give warning.
```
(config-if)# evpn  mh es-id

%Specify local ES ID
```

Reply:
Then documentation change is wrong. Need to be `evpn mh es-id <(1-16777215)|ESINAME>` (Notice `<>`, not `[]`).

Reply:
I meant fixing only in documentation because no form should work without arguments as before.

Reply:
Done. Thanks.